crates/ghost_actor/Cargo.toml

```diff
@@ -14,6 +14,7 @@ repository = "https://github.com/holochain/lib3h"
 [dependencies]
 crossbeam-channel = "=0.3.8"
 detach = { version = "=0.0.10", path = "../detach" }
+lib3h_tracing = { version = "=0.0.1", path = "../tracing" }
```

Comment:
should this be lib3h tracing or should it be something more generic that can be used by core too?

Reply:
@neonphog I figure this can move out of lib3h and become `holochain_tracing` when it's time. Or lib3h_tracing could even just be a wrapper around the more general one. Either way there's not much to it at this point, and probably never will be, so the move should be easy

Reply:
:+1: